modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Blogs/IBlogFeatureRepository.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Volo.Abp.Domain.Repositories;
+
+namespace Volo.CmsKit.Blogs
+{
+    public interface IBlogFeatureRepository : IBasicRepository<BlogFeature, Guid>
+    {
+        Task<List<BlogFeature>> GetListAsync(Guid blogId);
+
+        Task<List<BlogFeature>> GetListAsync(Guid blogId, string[] featureNames);
```

Comment:
Let use `List<string>` if it is not necessary to be array.

Reply:
I always try to use more primitive types when List functions are not required. But I've converted to List instead of array now 👍